### DISCUSSION TITLE
How to convert text scanned image in gray to 1bit black-white image?

### DISCUSSION THREAD

**Discussion by S90pngprgm** [ImageMagick/ImageMagick#5794] [Help]
The input is a scanned image from a text document.

I use the following command to make each 1x1 pixel into 3x3 pixel and snap the gray input image into black-white image. But the result is jagged. The attachment is an example.

```
convert in.jpg -scale 300% -negate -lat 50x50+10% -negate -type bilevel -monochrome -compress LZW out.tif
```

How to adjust the command to make the result not jagged?

![Untitled](https://user-images.githubusercontent.com/100811065/203463841-fdc8245c-9a78-49cb-ab11-16477fffdf21.png)

**Comment by snibgo** [comment]
You don't show the input or a desired output so advice isn't easy. But I suggest using "-resize" instead of "-scale", because that is where the large jaggies come from.

**Reply by S90pngprgm** [reply]
Indeed. Once I change -scale to -resize, the image is much less jagged. Thanks.
